README.md

```diff
@@ -141,6 +164,7 @@ export interface CreateIframeOptions {
 - `embeddingOrgId` (REQUIRED): The id string for your organisation that was issues to you by GetSetUp. Used by GetSetUp to tailor content to your organisation.
 - `deviceId` (OPTIONAL): This is an optional id used for analytics that will be reported back to your organisation.
 - `disableChat` (OPTIONAL): This is an optional flag that will disable the chat tab on the `joinClass` page. If this is true then the `tokenRequestCallBack` is not required.
+- `disableHelp` (OPTIONAL): This is an optional flag that will disable the help link chat on the `joinClass` page.
```

Comment:
`help link chat` sounds confusing to me - I don't understand why `chat` is mentioned. Maybe `help button`?

Reply:
Yeah, typo. Will fix. Thanks 👍